src/ui/Views/Course/Shared/_AboutCourseOverview.cshtml

```diff
@@ -10,40 +10,47 @@
-      How school placements work
-    </dt>
-    <dd asp-for="HowSchoolPlacementsWork" class="course-parts__fields__value @(string.IsNullOrWhiteSpace(courseEnrichment.HowSchoolPlacementsWork) ? "course-parts__fields__value--empty" : "")">
-      @if (school_placement) {
-        <a href="@url#HowSchoolPlacementsWork-wrapper" class="govuk-link">
-          <span asp-validation-for="HowSchoolPlacementsWork" >@ViewData.ModelState["HowSchoolPlacementsWork"].Errors.First().ErrorMessage</span></a>
-      }
-      @courseEnrichment?.HowSchoolPlacementsWork.DisplayText()
-    </dd>
+<div class="course-parts__item course-parts__item--borderless">
+  <h3 class="course-parts__title">
+    <a href="@url">About this course</a>
+  </h3>
+  <dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key @(about_course ? "course-parts__fields__label--error" : "")" id="section-AboutCourse">
+        About this course
+      </dt>
+      <dd asp-for="AboutCourse" class="govuk-summary-list__value @(string.IsNullOrWhiteSpace(courseEnrichment.AboutCourse) ? "course-parts__fields__value--empty" : "")">
+        @if (about_course)
+        {
+          <a href="@url#AboutCourse-wrapper" class="govuk-link"><span asp-validation-for="AboutCourse" ></span></a>
+        }
+        @courseEnrichment?.AboutCourse.DisplayText()
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key @(interview_process ? "course-parts__fields__label--error" : "")" id="section-InterviewProcess">
+        Interview process (optional)
+      </dt>
+      <dd class="govuk-summary-list__value @(string.IsNullOrWhiteSpace(courseEnrichment.InterviewProcess) ? "course-parts__fields__value--empty" : "")">
+        @if (interview_process)
+        {
+          <a href="@url#InterviewProcess-wrapper" class="govuk-link"><span asp-validation-for="InterviewProcess" ></span></a>
+        }
+        @courseEnrichment?.InterviewProcess.DisplayText()
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key @(school_placement ? "course-parts__fields__label--error" : "")" id="section-HowSchoolPlacementsWork">
+        How school placements work
+      </dt>
+      <dd asp-for="HowSchoolPlacementsWork" class="govuk-summary-list__value @(string.IsNullOrWhiteSpace(courseEnrichment.HowSchoolPlacementsWork) ? "course-parts__fields__value--empty" : "")">
+        @if (school_placement)
+        {
+          <a href="@url#HowSchoolPlacementsWork-wrapper" class="govuk-link">
+            <span asp-validation-for="HowSchoolPlacementsWork" >@ViewData.ModelState["HowSchoolPlacementsWork"].Errors.First().ErrorMessage</span></a>
+        }
+        @courseEnrichment?.HowSchoolPlacementsWork.DisplayText()
+      </dd>
+    </div>
   </dl>
-  @await Html.PartialAsync("CopyCourseNotice")
```

Comment:
Good to remove this @fofr?

It's the thing at the bottom here:

![screen shot 2019-01-16 at 17 56 01](https://user-images.githubusercontent.com/1650875/51268426-04664980-19b8-11e9-9f6e-db9fea6edaac.png)

Reply:
Not sure. It needs new text if it stays, that's for sure.